src/Symfony/Component/Mailer/Bridge/Mailgun/Smtp/MailgunTransport.php

```diff
@@ -22,9 +22,9 @@
  */
 class MailgunTransport extends EsmtpTransport
 {
-    public function __construct(string $username, string $password, EventDispatcherInterface $dispatcher = null, LoggerInterface $logger = null)
+    public function __construct(string $username, string $password, string $region = null, EventDispatcherInterface $dispatcher = null, LoggerInterface $logger = null)
     {
-        parent::__construct('smtp.mailgun.org', 465, 'ssl', null, $dispatcher, $logger);
+        parent::__construct('us' !== ($region ?: 'us') ? \sprintf('smtp.%s.mailgun.org', $region) : 'smtp.mailgun.org', 465, 'ssl', null, $dispatcher, $logger);
```

Comment:
No need to prefix `sprintf` with `\`

Reply:
Where can we find list of all functions that requires prefix (like `array_key_exists`)?

Reply:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/3048
but the best it to never use then when writing by hand, then ask php-cs-fixer to add them

Reply:
copy/paste from SesTransport which also use `\sprintf`. 

Note: The code contains 7 occurences of `\sprintf` shall we open a PR to remove them? 
Is there a way for fabbot to remove theme?

Reply:
Just remove the backslashes you added. I've just changed the configuration of PHP-CS-Fixer to take that into account and I've fixed everything from 3.4 to master.

Reply:
done